content/en/docs/reference/instrumentation/zpages.md

```diff
@@ -48,13 +50,57 @@ Up: 0 hr 00 min 16 sec
 Go version: go1.23.2
 Binary version: 1.32.0-alpha.0.1484&#43;5eeac4f21a491b-dirty
 Emulation version: 1.32.0-alpha.0.1484
+Paths: /healthz /livez /metrics /readyz /statusz /version
 ```
 
+#### structured response
+
+{{< feature-state for_k8s_version="v1.35" state="alpha" >}}
+
+Starting with Kubernetes v1.35, the `/statusz` endpoint supports a structured, versioned response format when requested with the appropriate `Accept` header. Without an `Accept` header, the endpoint returns the plain text response format by default.
+
+To request the structured response, use:
+```
+Accept: application/json;v=v1alpha1;g=config.k8s.io;as=Statusz
```

Comment:
Would `Accept: application/json` work?

Reply:
no, the `v`, `g`, `as` params are required.